### PR TITLE
Database ,week1 homework is ready for comments

### PR DESCRIPTION
**Pull request by aijazraja**

### CODE REVIEW

File: databases/week1/week1.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name= 'Done';
+
+-- Find all the tasks that are not marked as done
+
+SELECT task.title,status.name
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name= 'Not started' OR status.name= 'In progress' ;
```
Comment (pritalpatel) [inline databases/week1/week1.sql]:
```suggestion
SELECT task.title,status.name
FROM task, status
INNER JOIN status ON task.status_id = status.id
WHERE status.name= 'Not started' OR status.name= 'In progress' ;
```

File: databases/week1/week1.sql [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+select title ,due_date
+from task
+where description like '%database%'  or  title like '%database%' ;
+
+-- Get the title and status (as text) of all tasks
+
+SELECT task.title AS text, status.name AS text
+from task
+inner join status on task.status_id=status.id;
```
Comment (pritalpatel) [inline databases/week1/week1.sql:52]:
```suggestion
SELECT task.title AS text, status.name AS text
from task, status
inner join status on task.status_id=status.id;
```

**Review by pritalpatel — APPROVED** [review]
You can also put the query result in a comment.

File: databases/week1/week1.sql (hunk 2 above)
Comment (aijazraja) [reply databases/week1/week1.sql:52]: Hi Prital
i am getting this error
Error Code: 1066. Not unique table/alias: 'status'

File: databases/week1/week1.sql (hunk 1 above)
Comment (aijazraja) [reply databases/week1/week1.sql]: Hi Prital
i am getting this error
Error Code: 1066. Not unique table/alias: 'status'

**Pull request merged**